src/data.js

```diff
@@ -5160,6 +5160,27 @@ module.exports = [
       'CSS',
     ],
   },
+  {
+    name: 'Alex Kiss',
+    description:
+      'SRE/DevOps - just a geek in general',
+    url: 'https://sysrex.tech/uses/',
```

Comment:
I get a ERR_NAME_NOT_RESOLVED error there.
Does the domain exists?

Reply:
sorry I was missing the www. -> I see that the other CNAME I forgot to add.